modules/network-observability-enriched-flows.adoc

```diff
@@ -32,8 +32,16 @@ spec:
         topic: netobserv-flows-export   <1>
         tls: 
           enable: false                 <2>
+     - type: IPFIX                      <3>
```

Comment:
```suggestion
  - type: IPFIX                      <3>
```

Indentation is wrong here ? maybe also on the arguments below

Reply:
Addressed for all